grammar.js

```diff
@@ -1,5 +1,5 @@
-const DIGITS = token(sep1(/[0-9]+/, /_+/))
-const HEX_DIGITS = token(sep1(/[A-Fa-f0-9]+/, '_'))
+const DIGITS = token(sep1(/[0-9]+/, /_+/));
```

Comment:
unrelated.
All those semicolons, space change, comma, ... all of that is unrelated.

Reply:
removed now